gameoflife.py

```diff
@@ -76,25 +95,25 @@ def print_board(board):
         print('|')
 
 
-def main(args):
-    if len(args) != 2:
-        print("Please enter 2 arguments\n\t-file path to initial board state\n\t-number of generations")
-        exit()
+def main(args : argparse.Namespace) ->None:
     try:
-        board = init_board(args[0])
-        generations = int(args[1])
-    except ValueError:
-        print(f"2nd argument must be a number, '{args[1]}' was entered.")
-        exit()
+        board = init_board(args.file_path)
     except Exception as e:
         print(e)
         exit()
```

Comment:
you can remove the try-except block, because all it does is print the error and terminate the program, which will happen anyway.

Reply:
This way the error would be printed with a lot of additional information. is that what you meant or is there a way to print only the error without except?

Reply:
please attach a screenshot of how it looks now

Reply:
please add a screenshot of how it looks now

Reply:
<img width="966" alt="Screen Shot 2023-03-07 at 12 31 16" src="https://user-images.githubusercontent.com/110101070/223397111-60aa2ac1-e213-46fe-b713-7db1b125617a.png">

Reply:
Yes it's good enough